htmlelements-matchers/src/main/java/ru/yandex/qatools/htmlelements/matchers/decorators/ConditionMatcherDecorator.java

```diff
@@ -22,16 +25,22 @@ public ConditionMatcherDecorator(Matcher<? super T> matcher, E itemToMatchCondit
     protected boolean matchesSafely(T item) {
         if (condition.matches(itemToMatchCondition)) {
             return matcher.matches(item);
+        } else {
```

Comment:
Of course this does the exact same thing, this was just for slight readability improvement, where the two exit points from the method are aligned horizontally.